docs/index.md

```diff
@@ -53,10 +83,19 @@ This project was initially inspired by a [gist](https://gist.github.com/mosheesh
 
 ## Contributing
 
-See [CONTRIBUTING](https://github.com/testcontainers/testcontainers-java/blob/master/CONTRIBUTING.md).
+* Star the project on [Github](https://github.com/testcontainers/testcontainers-java) and help spread the word :)
+* Join our Slack: http://slack.testcontainers.org
+* [Post an issue](https://github.com/testcontainers/testcontainers-java/issues) if you find any bugs
+* Contribute improvements or fixes using a [Pull Request](https://github.com/testcontainers/testcontainers-java/pulls). If you're going to contribute, thank you! Please just be sure to:
+    * discuss with the authors on an issue ticket prior to doing anything big
+    * follow the style, naming and structure conventions of the rest of the project
+    * make commits atomic and easy to merge
+    * when updating documentation, please see [our guidance for documentation contributions](contributing_docs.md).
+    * verify all tests are passing. Build the project with `./gradlew check` to do this.
```

Comment:
rendered, `NB` is at the same line as "verify all...". WDYT about adding `  ` (double space) to force it to the next line?

Reply:
I thought it was deliberate, as it's about running the Gradle build...

Reply:
TBH I didn't even know what NB means :confounded: